tests/toranj/test-013-off-mesh-route-traffic.py

```diff
@@ -132,7 +134,23 @@
 fed1.add_route(OFF_MESH_ROUTE_3)
 fed1.add_ip6_address_on_interface(OFF_MESH_ADDR_3)
 
-time.sleep(0.5)
+# Wait till network data is updated on all nodes and they all see all
+# the added off-mesh routes.
+
+
+def check_off_mesh_routes():
+    verify(
+        len(wpan.parse_list(r1.get(wpan.WPAN_THREAD_OFF_MESH_ROUTES))) ==
+        NUM_ROUTES + 1)
+    verify(
+        len(wpan.parse_list(r2.get(wpan.WPAN_THREAD_OFF_MESH_ROUTES))) ==
+        NUM_ROUTES + 1)
+    verify(
+        len(wpan.parse_list(sed2.get(wpan.WPAN_THREAD_OFF_MESH_ROUTES))) ==
+        NUM_ROUTES)
```

Comment:
The above checks have `NUM_ROUTES + 1`, why not here?

Reply:
If a node itself adds a route, the route entry will be seen twice in `"Thread:OffMeshRoutes"` wpantund list (one time as part of leader/network-wide network data and again as part of the local network data ("IsLocal" flag is set)). The `+1` is to account for that.  In the test network we add off-mesh routes on `r1`, `r2` and `fed` but not on `sed1`. So when things are synced and stable, `r1 `and `r2` should see 4 entries in their list while `sed` should only see `3`.

Reply:
How about having a named value like NUM_ROUTES_LOCAL instead of magic number.